curve25519-dalek/src/montgomery.rs

```diff
@@ -394,6 +391,25 @@ impl<'a, 'b> Mul<&'b Scalar> for &'a MontgomeryPoint {
 
         x0.as_affine()
     }
+
+    /// Given `self` \\( = u\_0(P) \\), and a big-endian bit representation of an integer
+    /// \\(n\\), return \\( u\_0(\[n\]P) \\).
+    #[cfg(feature = "hazmat")]
+    fn mul_bits_be(self, bits: impl Iterator<Item = bool>) -> MontgomeryPoint {
+        self.mul_bits_be(bits)
+    }
+}
+
+/// Multiply this `MontgomeryPoint` by a `Scalar`.
+impl<'a, 'b> Mul<&'b Scalar> for &'a MontgomeryPoint {
```

Comment:
I think all of these lifetimes can be elided?
```suggestion
impl Mul<&Scalar> for &MontgomeryPoint {
```